.gitmodules

```diff
@@ -23,11 +23,10 @@
 	url = https://github.com/NOAA-GSL/MPASSIT.git
         branch = gsl/develop
         ignore = all
-[submodule "sorc/RRFS_UTILS"]
-	path = sorc/RRFS_UTILS
-	url = https://github.com/hu5970/RRFS_UTILS.git
-        ignore = all
 [submodule "ush/wxflow"]
 	path = ush/wxflow
 	url = https://github.com/NOAA-EMC/wxflow.git
         ignore = all
+[submodule "sorc/RRFS_UTILS"]
+	path = sorc/RRFS_UTILS
+	url = https://github.com/NOAA-GSL/RRFS_UTILS
```

Comment:
@ShawnMurdzek-NOAA Thanks a lot for this PR!
Sorry for a pick comment. Could you modify the `RRFS_UTILS` repo in its original location (i.e. before `ush/wxflow`)? (Also, we want to keep the original `ignore = all` line to speedup normal git operations). Thanks!